tesseract_motion_planners/ompl/src/continuous_motion_validator.cpp

```diff
@@ -139,8 +138,8 @@ bool ContinuousMotionValidator::continuousCollisionCheck(const ompl::base::State
   }
   mutex_.unlock();
 
-  Eigen::Map<Eigen::VectorXd> start_joints = extractor_(s1);
-  Eigen::Map<Eigen::VectorXd> finish_joints = extractor_(s2);
+  Eigen::Map<Eigen::VectorXd> start_joints = fromRealVectorStateSpace(s1, si_->getStateDimension());
+  Eigen::Map<Eigen::VectorXd> finish_joints = fromRealVectorStateSpace(s2, si_->getStateDimension());
```

Comment:
How would this work with a constrained state space? The intent behind the extract was to avoid multiple implementations of this class.

Reply:
Check out 74d7dda, 8cbdbce, 27f376e, and 792626f, which add a constrained planner profile that inherits from the existing RVSS plan profile with minimal additional code. It at least compiles on my machine, but I haven't tested it yet.

I think we still need the concept of an `OMPLStateExtractor`, but it only applies to real vector state spaces, not all state spaces in general.